tests/unit/DetailsList.spec.ts

```diff
@@ -93,25 +92,25 @@ describe('Details List', () => {
     ]
   }
 
-  it('Displays nothing if no comments are present in the filing', () => {
+  it('Displays no details if filing contains no comments', () => {
     const wrapper = shallowMount(DetailsList, { store, propsData: { filing: mockNoCommentsFiling } })
     const vm: any = wrapper.vm
 
-    expect(vm.$el.querySelector('.details-list')).toBeNull()
+    expect(vm.$el.querySelector('.detail-body')).toBeNull()
```

Comment:
Should this file be updated to use wrapper.find() as well?

Reply:
It could be :)  I was trying to change as little as possible in existing code. What do you think?

Reply:
It was quick so I did it :)